web/modules/custom/dpl_event/dpl_event.deploy.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @file
+ * Event deploy hooks.
+ *
+ * These get run AFTER config-import.
+ */
+
+use Drupal\Core\Entity\FieldableEntityInterface;
+
+/**
+ * Pre-populate data to new non-WYSIWYG field field_description.
+ *
+ * We also empty out the old field. The field has been set to be hidden
+ * from the editors, and we'll delete it in a future deploy.
+ */
+function dpl_event_deploy_port_description_fields() : string {
+  $message = _dpl_event_port_wysiwyg('eventseries', 'field_event_description', 'field_description');
+  $message .= _dpl_event_port_wysiwyg('eventinstance', 'field_event_description', 'field_description');
+
+  return $message;
+}
+
+/**
+ * A helper function, for porting WYSIWYG fields to plain textareas.
+ */
+function _dpl_event_port_wysiwyg(string $entity_type, string $source_field, string $target_field): string {
```

Comment:
Such reusable helper functions are nice but placing it in the `dpl_event` module makes it hard to reuse in other modules.

In the future we should consider moving it to somewhere shareable - perhaps in the profile?

Reply:
Right now, i made it as a helper function as i needed to do it on two fields.

If this is something we end up needing again, we can always move it out of dpl_event